packages/shared-core/src/filters.ts

```diff
@@ -368,13 +504,15 @@ export const buildQuery = (filter: SearchFilter[]) => {
       value = `${value}`?.toLowerCase() === "true"
     }
     if (
-      ["contains", "notContains", "containsAny"].includes(operator) &&
+      ["contains", "notContains", "containsAny"].includes(
+        operator.toLocaleString()
+      ) &&
       type === "array" &&
       typeof value === "string"
     ) {
       value = value.split(",")
     }
-    if (operator.startsWith("range") && query.range) {
+    if (operator.toLocaleString().startsWith("range") && query.range) {
```

Comment:
How come `toLocaleString` is needed here?

Reply:
Again unsure, but would prefer not to remove something that may have a very explicit purpose - it needs a string and perhaps there are things in the JS that configure the `operator` as a number etc.